### PR TITLE
add hyprland-per-window-layout (rust)

### PR DESCRIPTION
**Pull request by coffebar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,7 +128,8 @@ Awesome list for Hyprland, that includes useful tools and libraries that either
 
 ### Input
 
-- [Hyprkeys](https://github.com/hyprland-community/Hyprkeys) ![go][go] (Utility for dealing with keybinds in Hyprland)
+- [Hyprkeys](https://github.com/hyprland-community/Hyprkeys) ![go][go] (Utility for dealing with keybinds in Hyprland
```
Comment (yavko) [inline README.md:131]: Why? What did the `)` do to you to deserve this?!??!

File: README.md [hunk 2]
```diff
@@ -128,7 +128,8 @@ Awesome list for Hyprland, that includes useful tools and libraries that either
...
+- [Hyprland per window layout](https://github.com/coffebar/hyprland-per-window-layout) ![rust][rs] (Per window keyboard layout, zero-configuration, just works out of the box)
 - [hyprland-per-window-layout](https://github.com/MahouShoujoMivutilde/hyprland-per-window-layout) ![shell][sh] (Script to maintain per window keyboard layout) (language)
```
Comment (yavko) [inline README.md:133]: Plz don't remove things, instead mark as deprecated or whatever.

File: README.md [hunk 3]
```diff
@@ -128,7 +128,8 @@ Awesome list for Hyprland, that includes useful tools and libraries that either
 
 ### Input
 
-- [Hyprkeys](https://github.com/hyprland-community/Hyprkeys) ![go][go] (Utility for dealing with keybinds in Hyprland)
+- [Hyprkeys](https://github.com/hyprland-community/Hyprkeys) ![go][go] (Utility for dealing with keybinds in Hyprland
+- [Hyprland per window layout](https://github.com/coffebar/hyprland-per-window-layout) ![rust][rs] (Per window keyboard layout, zero-configuration, just works out of the box)
```
Comment (yavko) [inline README.md:132]: This casing is ugly, makes me feel iky

**Review by yavko — CHANGES_REQUESTED** [review]
Plz fix, thx

File: README.md (hunk 1 above)
Comment (coffebar) [reply README.md]: Sorry, I didn't mean it.  The visual editor showed that everything was in place ¯\(°_o)/¯

Returned  `)` back

**Review by yavko — APPROVED** [review]

**Pull request merged**